### DISCUSSION TITLE
Screen responsiveness on different devices.

### DISCUSSION THREAD

**Discussion by Jooaomar** [bvaughn/react-resizable-panels#354] [Q&A] [answered]
it would be correct to use this lib to handle screen responsiveness. Does anyone have an example?

**Comment by bvaughn** [comment, marked as answer]
This library doesn't have anything to do with screen size or responsiveness. Only resizing.

**Reply by bvaughn** [reply]
I'm going to close this because I think this question is basically outside of the scope of this project.

**Reply by Jooaomar** [reply]
Thank you, that's exactly what I wanted to know.
